pmsv2/params.go

```diff
@@ -0,0 +1,109 @@
+package pmsv2
+
+import (
+	"time"
+
+	"github.com/xendit/xendit-go/pmsv2/card"
+	"github.com/xendit/xendit-go/pmsv2/constant"
+	"github.com/xendit/xendit-go/pmsv2/directdebit"
+	"github.com/xendit/xendit-go/pmsv2/ewallet"
+	"github.com/xendit/xendit-go/pmsv2/overthecounter"
+	"github.com/xendit/xendit-go/pmsv2/qrcode"
+	"github.com/xendit/xendit-go/pmsv2/virtualaccount"
+)
+
+type CreatePaymentMethodParams struct {
+	Type               constant.PaymentMethodTypeEnum `json:"type"`
+	Country            constant.CountryEnum           `json:"country"`
+	CustomerID         *string                        `json:"customer_id"`
+	ReferenceID        string                         `json:"reference_id,omitempty"`
+	Reusability        constant.ReusabilityEnum       `json:"reusability"`
+	Description        *string                        `json:"description"`
+	Metadata           map[string]interface{}         `json:"metadata"`
+	BillingInformation *card.BillingInformation       `json:"billing_information"`
+
+	Card           *card.CreateMethod           `json:"card"`
+	DirectDebit    *directdebit.CreateMethod    `json:"direct_debit"`
+	Ewallet        *ewallet.CreateMethod        `json:"ewallet"`
+	OverTheCounter *overthecounter.CreateMethod `json:"over_the_counter"`
+	VirtualAccount *virtualaccount.CreateMethod `json:"virtual_account"`
+	QRCode         *qrcode.CreateMethod         `json:"qr_code"`
+
+	ForUserID      string `json:"-"`
+	IdempotencyKey string `json:"-"`
+}
+
+type ValidateOTPRequest struct {
+	ID   string `param:"id"`
+	Code string `json:"auth_code"`
+
+	ForUserID      string `json:"-"`
+	IdempotencyKey string `json:"-"`
+}
+
+type ExpireRequest struct {
+	ID                     string  `json:"-"`
+	UnlinkSuccessReturnURL *string `json:"unlink_success_return_url"`
```

Comment:
please remove this for now, not live yet

Reply:
done